RemixHackathon.md

```diff
@@ -0,0 +1,119 @@
+# ETHBerlin Remix hackathon
+
+First please join https://gitter.im/ethereum/remix-dev-plugin ;)
+
+Welcome to this Remix hackathon!
+We are really happy to see new people involved in improving tools ;)
+
+This document aims to give more understanding about this hackathon.
+
+Important! You don't need to be a killer blockchain dev for participating, just being a curious developer and have a basic understanding of Ethereum tech.
+
+## Introduction
+
+Remix is a suite of tools written in JavaScript used for developing smart contract,
+It contains the following packages:
+
+https://github.com/ethereum/remix
+ - remix-lib
+ - remix-debug (contract debugging)
+ - remix-analysis (solidity static analysis)
+ - remix-solidity (solidity compiler)
+ - remix-tests (smart contract unit testing)
+
+https://github.com/ethereum/remix-ide
+ - remix-ide (IDE - remix.ethereum.org)
+
+https://github.com/ethereum/remixd
+ - remixd (accessing local files from remix IDE)
+
+`remix.ethereum.org` host remix IDE web application, used to write, analyze, test and debug Ethereum smart contracts.
+
+We have recently published an update that provide a way to develop external plugin.
+
+The purpose of this hackathon is to build plugins for Remix.
+
+## General overview
+
+Here we describe basic information about this event.
+
+ - Each team should have at least one person present at the event.
+ - Each team should choose a plugin from the plugin list.
```

Comment:
perhaps link plugin list to https://github.com/ethberlin-hackathon/ETHBerlin-Teambuilding/issues?q=is%3Aissue+is%3Aopen+label%3A%22Remix-IDE+%3Adesktop_computer%3A%22